vip-feed-cache/class-vip-go-feed-cache.php

```diff
@@ -11,7 +11,7 @@ class VIP_Go_Feed_Cache extends SimplePie_Cache {
 	 * @param string $extension 'spi' or 'spc'.
 	 * @return VIP_Go_Feed_Cache_Transient Feed cache handler object that uses transients and normalizes SimplePie Build number.
 	 */
-	public function create( $location, $filename, $extension ) {
+	public static function get_handler( $location, $filename, $extension ) {
```

Comment:
Technically a backwards compatibility break if somebody were using this class/method directly.

Generally would be best to leave the existing method and show a deprecation notice/warning so we're more prepared if somebody is extending/using it, rather than causing fatals upon deploy.

Reply:
@WPprodigy When I grepped for it, I didn't find any references, so I figured we should be a-ok?

Reply:
Could be tricky to cache if some custom code was using the abstraction layers and stuff though, or the `wp_feed_options` hook for example 😬 

Low probability, but definitely gotta be careful when removing public callables.